### PR TITLE
sky: initial build 11.x

### PR DESCRIPTION
**Pull request by miju12**

### CODE REVIEW

File: sky.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      "device": "Redmi 12 5G/Poco M6 Pro 5G",
+      "filename": "crDroidAndroid-15.0-20241111-sky-v11.0-BETA.zip",
+      "download": "https://sourceforge.net/projects/crdroid/files/sky/11.x/crDroidAndroid-15.0-20241111-sky-v11.0-BETA.zip/download",
+      "timestamp": 1731343212,
+      "md5": "a145a35fb8f908d68e6eb47835a8f05e",
+      "sha256": "ec7598bd487de5b5f46fdcb31c65b923fcd86055705c245a623826f691cacb3a",
+      "size": 1205184829,
+      "version": "11.0",
+      "buildtype": "BETA",
+      "forum": "https://xdaforums.com/t/rom-15-redmi-12-5g-poco-m6-pro-5g-crdroid-v11.4702481/",
```
Comment (gwolf2u) [inline sky.json:15]: fix forum post please
missing follow back links to site (footer)

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: sky.json (hunk 1 above)
Comment (miju12) [reply sky.json:15]: Fixed now

Comment (gwolf2u) [reply sky.json:15]: Still bad
Do check forum template
https://github.com/crdroidandroid/crdroid_features/blob/15.0/ForumPostTemplate
Also update features url

**Comment by gwolf2u** [comment]
inactive

**Pull request closed without merging**